.travis.yml

```diff
@@ -53,8 +51,5 @@ before_cache:
 cache:
   timeout: 1000
   directories:
-  - $HOME/.npm-cache
-  - $HOME/.npm-prefix
-  - metron-interface/metron-config/node_modules
```

Comment:
Migrated to Travis CI's own node/npm instance managed by nvm. The above configs no longer needed.